control_parameters: {
  pid_parameters: {Kd: 0.0, Ki: 0.02, Kp: 0.04},
  power: {max: 2.0, min: 0},
  step: {power: 2, trigger: 2},
  manual: {power: 2.0},
  automatic: {width: 105, time: 10, mode: 0}
}
